components/card/index.js

```diff
@@ -58,6 +58,17 @@ const Card = ({ data }) => {
                   <For of={item.examples} render={example => <p key={example}>{example}</p>} />
                 </Choose.When>
               </Choose>
+              <h3>Sinônimos:</h3>
+              <Choose>
+                <Choose.When condition={item.synonyms === undefined}>
```

Comment:
I think this condition can be:
```js
item.synonyms == null // or undefined
```
Because on JavaScript `double equal` serves to verify `null` and `undefined`, but just for these cases!

Reply:
I understand. The solution can be replace `item.synonyms === undefined` to `item.synonyms === undefined or item.synonyms === null`. Do you agree? Because if synonyms field does not exists in item, it will be `undefined`, but, the field synonyms can be `null` too.

Reply:
`item.synoyms == undefined` or `item.synoyms == null`, because this condtion form verify if undefined or null at the same time! The secret is  `==` double equal.

Reply:
Ahh, I'm sorry. I did know. I'll change this line. Thanks!

Reply:
Changed!